webapps/text-labeling/backend.py

```diff
@@ -0,0 +1,14 @@
+import dataiku
+from dataiku.customwebapp import get_webapp_config
+
+from lal.api import define_endpoints
+from lal.app_configuration import prepare_datasets
+from lal.classifiers.text_classifier import TextClassifier
+from lal.handlers.dataiku_lal_handler import DataikuLALHandler
+
+config = get_webapp_config()
+prepare_datasets()
+initial_df = dataiku.Dataset(config["unlabeled"]).get_dataframe()
+queries_df = dataiku.Dataset(config["queries_ds"]).get_dataframe() if "queries_ds" in config else None
```

Comment:
Cannot we keep it in case we want to use AL later ?

Reply:
I'd prefer to keep it clean and remove queries for now (just pass `None` to `define_endpoints`). Usually, things that you leave for a little while to be used later tend to stay forever and make the code harder to understand. Since there are examples of using queries in the other apps it'll be easy to add it when it's needed

Reply:
Definitely true, I always tend to think on future and sometimes that is a bad habit. Fixed in 3344cae